ci/scripts/benchmark.sh

```diff
@@ -45,38 +59,63 @@ function cleanup {
 
 trap cleanup EXIT
 
+DB_USER=dbuser
+DB_PWD=dbpwd
```

Comment:
This may leak credentials, use github tokens or sth similar.

Reply:
This is just used to run daily,  and the instance will be destroyed when the pipeline ends.